src/app/legacy/containers/ConsentBanner/Banner/cookie.amp.jsx

```diff
@@ -129,6 +118,16 @@ const OptionsList = styled.ul`
   }
 `;
 
+/* Custom hover and focus indicator styling applied to pseudo-element. Global focus indicator styling has been removed. */
```

Comment:
@Isabella-Mitchell helped with this bit so may be better able to answer/can check mine is correct but im pretty sure this is so that the focus border of the icon sits on top of the banner div, so that the top and right side of it arent cut off? as it sits right in the corner of the div

Reply:
Yep that's correct, it's a similar approach we use to the Media Player button and Navigation Links.